### ISSUE TITLE
Dash-button with MAC '00:bb:3a:04:de:59' is not discovered

### ISSUE THREAD

**Issue by mwittig** [mwittig/pimatic-amazing-dash-button#3]
This popped up here: https://forum.pimatic.org/topic/2427/amazing-dash-button-one-of-two-buttons-not-recognized

The MAC does not the known vendor ids assigned to Amazon. Maybe a new one? To be investigated.

**Comment by mwittig** [comment]
Checked 'MAC Address Block Large (MA-L)' at https://regauth.standards.ieee.org/standards-ra-web/pub/view.html#registries. The vendor is is marked as being private as companies may deny public disclosure. Strange, why Amazon is starting with such bad habbits.

```
00-BB-3A   (hex)                Private
00BB3A     (base 16)        Private
```

**Comment by mwittig** [comment]
Released pimatic-amazing-dash-button@0.9.10

**Comment by kiwikern** [comment]
Fix worked, can now use my manually added device. Thanks a lot. 👍
